invenio_app_ils/ui/src/invenio_app_ils/common/components/ESSelector/ESSelectorModal.js

```diff
@@ -56,6 +56,7 @@ export default class ESSelectorModal extends Component {
                 ? this.props.saveButtonContent
                 : 'Save'
             }
+            title="Requires to select a patron"
```

Comment:
Since this modal is used everywhere we should pass a prop if you want to have a specific title since now when you choose e.g. a keyword it will display a message that a patron is required.